### ISSUE TITLE
can't get network statistics window

### ISSUE THREAD

**Issue by OSS542** [evilsocket/opensnitch#193]
I can't seem to find how to get a "network statistics" window to appear.  The opensnitchd daemon appears to run correctly as root, and opensnitch-ui also appears to run correctly.  There are no errors in the logs, and the connection block/allow dialog appears and works correctly.  Is there a seperate executable to be run to obtain "network statistics ?

**Comment by Northern-Lights** [comment]
Looks like that happens [here][1]

[`sni-qt`][3] is apparently supposed to bridge the [QSystemTrayIcon][2] to a message tray icon appropriate for GNOME's message center (I'm assuming you're using GNOME like me; I also cannot view the stats window.)  Maybe something changed with GNOME, and the icon is no longer able to be put where it belongs.  I know that they removed that little icon bar on the bottom left; I'm not sure if that's where it would have went.

[1]: https://github.com/evilsocket/opensnitch/blob/master/ui/opensnitch/service.py#L103
[2]: https://doc.qt.io/qt-5/qsystemtrayicon.html
[3]: https://askubuntu.com/questions/155545/how-do-i-get-qsystemtrayicon-to-show-up-in-gnome-3#158959

**Comment by OSS542** [comment]
Actually, I don't use a desktop at all.  The linux system I am using is a custom design of my own and is built entirely from source code.  It's much faster for me to use than a desktop.  It uses the i3 tiling window manager under Xorg displayed across multiple monitors.  I was hoping that there was some way of invoking the stats display which did not depend on using a desktop.

**Comment by Northern-Lights** [comment]
Agreed, I would like a global hotkey or something like that to invoke the stats dialog.  I'm not sure at this time how Qt plays with your specific WM, or if hotkeys (however they may be implemented in Qt) will work across all DEs or WMs.

**Comment by OSS542** [comment]
I see this was closed.   What was the reason ?

**Comment by evilsocket** [comment]
oh sorry my bad, misclicked, didn't intend to

**Comment by bodqhrohro** [comment]
Global hotkeys wouldn't generally work in Wayland-based environments anyway. The UI should be reworked to make possible launching certain windows via command line flags of `opensnitch-ui` or dedicated utilities, instead of Windoze-way approach of making all the control dependent on a tray icon, especially according the fact how diversely and unreliably do the tray icons work among GNU/Linux graphical environments.

**Comment by DanyGee** [comment]
Add global hotkey or add a terminal command to launch statistics window like **opensnitch-stats** or something similar.
